storage/Utils/Logger.cc

```diff
@@ -155,7 +132,24 @@ namespace storage
 	--count;
     }
 
-
     int Silencer::count = 0;
 
+
+    const string default_log_path()
+    {
+	static const string default_path = "/var/log/libstorage-ng.log";
```

Comment:
This is a typical one-liner. There is no need to declare the constant when all you ever do with it is return it in the next line of code. Please simply _return_ that string right here and now. Yes, it is a `const char []` technically, but since there is a std::string constructor for that, it will automatically be converted to a std::string.